### PR TITLE
Ksubrama/env fix

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

**Comment by ksubrama** [comment]
Use the path-helper code from core chef client to discover bootstrap templates.

@btm @jdm @smurawski

**Comment by jdm** [comment]
You want @jdmundrawala, not me.

**Comment by btm** [comment]
@ksubrama can you put context onto this PR? I forget what this fixes.

File: lib/knife-windows/path_helper.rb [hunk 1]
```diff
@@ -148,6 +148,83 @@ def self.escape_glob(*parts)
...
+          # HOMEDRIVE HOMEPATH
+          # HOMESHARE HOMEPATH
+          # USERPROFILE
+
+          paths << ENV['HOME']
+          paths << ENV['HOMEDRIVE'] + ENV['HOMEPATH'] if ENV['HOMEDRIVE'] && ENV['HOMEPATH']
+          paths << ENV['HOMESHARE'] + ENV['HOMEPATH'] if ENV['HOMESHARE'] && ENV['HOMEPATH']
+          paths << ENV['USERPROFILE']
+        end
+        paths << Dir.home
```
Comment (btm) [inline lib/knife-windows/path_helper.rb]: @ksubrama @jdmundrawala says this is bug.

**Comment by ksubrama** [comment]
This adds the "find the path to the home directory sensibly" helper code that was added to core chef to this plugin.  We copy-paste the code to allow it to work with Chef 11.  There was only one place where env home is being referenced in this plugin and I changed it to use the new helper instead.

File: lib/knife-windows/path_helper.rb (hunk 1 above)
Comment (ksubrama) [inline lib/knife-windows/path_helper.rb]: Fixed.

**Pull request merged**